### ISSUE TITLE
Add checksum to descriptor

### ISSUE THREAD

**Issue by Sjors** [ACINQ/phoenix#402]
I was able to import the swap-in wallet descriptor into a Bitcoin Core watch-only wallet. But I had to manually add the checksum.

https://github.com/bitcoin/bips/blob/master/bip-0380.mediawiki#user-content-Checksum

**Comment by t-bast** [comment]
Right, we initially didn't have the checksum code written yet, and didn't update to include it, but that should be easy to do.

**Comment by dpad85** [comment]
Fixed in https://github.com/ACINQ/lightning-kmp/pull/563.
